README.rst

```diff
@@ -168,33 +210,46 @@ A record is a dict with the "permissions" and "data" keys.
 
 .. code-block:: python
 
-    # You can pass a python dictionary to create the record
-    # bucket='default' can be omitted since it's the default value
+    # You can pass a python dictionary to create the record.
+    client.create_record(data={'status': 'done', title: 'Todo #1'},
+                         collection='todos', bucket='default')
 
-    client.create_record(data={'id': 1234, status: 'done', title: 'Todo #1'},
+    # You can use id to specify the record id when creating it.
+    client.create_record(id='todo2', data={'status': 'doing', 'title': 'Todo #2'},
                          collection='todos', bucket='default')
 
-    # Retrieve all records.
+    # Or get an existing one by its id.
+    record = client.get_record('todo2', collection='todos', bucket='default')
+
+    # Or Retrieve all records.
     records = client.get_records(collection='todos', bucket='default')
 
-    # Retrieve records timestamp.
+    # Or retrieve records timestamp.
     records_timestamp = client.get_records_timestamp(collection='todos', bucket='default')
 
-    # Retrieve a specific record and update it.
+    # To retrieve a specific record and update it.
     record = client.get_record('89881454-e4e9-4ef0-99a9-404d95900352',
                                collection='todos', bucket='default')
+    record['status'] = 'doing'
     client.update_record(record, collection='todos', bucket='default')
 
```

Comment:
I also suggest we should remove this part. There are better ways to do this (via patch). What do you think? I suggest just leave the reference for update_record( ) passing a record as a parameter.